scripts/mural.js

```diff
@@ -0,0 +1,16 @@
+function showMuralModalContainer() {
+    const muralModalContainer = document.getElementById('muralModalContainer')
```

Comment:
see comments below... these will have to be targeted by class (since there are multiple), and you'll probably want to use `document.querySelectorAll` to grab them.